docs/api/api/api.txt

```diff
@@ -1433,14 +1433,16 @@ XmlResult: buildinfo
 
 POST /trigger/runservice
 
-  This call needs a token created via "createtoken" command to identify user and
-  package which shall be updated via a source service. The token must be delivered
-  as HTTP header:
+  This call needs a token to identify user and package which shall be updated
+  via a source service. See the cmd=create in the /person/<userid>/token route.
+
+  The token itself must be delivered as HTTP header:
     Authorization: Token <TOKEN_STRING>
 
 Parameters
-  project: To be used together with "package" to identify the object.
-  package:
+  project: To be used together with "package" to identify the object. Must not be used
+           when the token is bound to a package.
+  package: see project parameter.
```

Comment:
What does this mean? That the package parameter must not be used in the same way the project parameter shall not be used when the token is bound to a package?

Reply:
You always need project and package parameter together. Or neither of them.

Reply:
(in future the project parameter alone may work though, but this is not implemented)

Reply:
> You always need project and package parameter together. Or neither of them.

Ah, ok. That part was not clear to me

Reply:
Just out of interest, what does happen if you pass the project & package parameters but the token is bound to a specific package?

Reply:
an error is reported IIRC